package.json

```diff
@@ -1,4 +1,6 @@
 {
+  "name": "dnssec-oracle",
```

Comment:
scope to ens domains. So ```@ensdomains/dnssec-oracle```

Reply:
Just a generic question. Currently `dnssec-oracle` is owned by my account. How do we allow anyone from our team to publish under `@ensdomains/dnssec-oracle`?

Reply:
@makoto I will need to add you as an admin to the ensdomains npm organization and then you can publish it public